modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -18,54 +31,70 @@ The following table contains examples of the URLs you need to supply, depending
 [NOTE]
 Anypoint Platform makes an HTTP call to the configured OIDC endpoints, and if there is no response after five seconds, the HTTP call times out.
-* *Token Introspection Client*
- ** *Client ID*: The client ID for an existing client in your IdP capable of introspection of _all_ tokens from all clients.
-  *** For Okta, this value should be a "Confidential" client.
-  *** For ForgeRock, this value should be a "Confidential" client.
-  *** For Salesforce, this value should be a "Confidential" client.
- ** *Client Secret*: The client secret that corresponds to the client ID.
-* *OpenID Connect Authorization URLs*
- ** *Authorize URL*: The URL where the user authenticates and grants OpenID Connect client applications access to the user's identity.
- ** *Token URL*: The URL that provides the user’s identity, encoded in a secure JSON Web Token.
++
+.. Select *Advanced Settings* for these options:
+* *Disable server certificate validation*
++
+Disables server certificate validation if your OpenID client management instance presents a self-signed certificate, or one signed by an internal certificate authority.
+* *Enable client deletion in Anypoint Platform*
++
+Enables deletion of clients created with this integration.
+* *Enable client deletion and updates in IdP*
++
+To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`. The `clientRegistrationUrl` is the value you configured in *Client Registration URL*. For example: `+https://identity.example.com/oauth2/connect/register/6779ef20e75817b79602+`. 
+. In *Authentication Method*, complete the information for the type of authentication to use for this client provider:
+* *Authorization Header*
++
+The HTTP authorization header provides a static token that doesn't expire to authenticate with a server. This header is required if the provider restricts registration requests to authorized clients. If you prefer to use short-lived bearer access tokens, use the OAuth 2.0 private key option instead. 
++
+[%autowidth.spread]
+|===
+|Client Provider | Value
+| Okta | `SSWS ${api_token}`, where `api_token` is an API token created through Okta.
+| ForgeRock | `Bearer ${api_token}`, where `api_token` is an API token created through ForgeRock.
+| Salesforce | `Bearer ${api_token}`, where `api_token` is an API token created through Salesforce.
+|===
+.. Select *Use OAuth2 private key jwt* to use a private key instead of and authorization header, and provide these values:
+* *Token Client ID*
++
+Create a JWT (JSON Web Token) and sign it using the private key. Use this as the client assertion when making the `/token` endpoint API call. 
```

Comment:
ah my bad! I only focused on JWT thing, but this field isn't about the JWT. Sorry!

```suggestions
This is the unique identifier of the application (i.e. client ID) registered within your IDP. It is used to identify the calling application when requesting a bearer token from the Anypoint platform during client registration. 

```